usr/sbin/p11sak/p11sak.mk

```diff
@@ -1,13 +1,13 @@
-sbin_PROGRAMS += usr/sbin/p11sak_tool/p11sak
-noinst_HEADERS += usr/sbin/p11sak_tool/p11sak.h
+sbin_PROGRAMS += usr/sbin/p11sak/p11sak
```

Comment:
I would squash this commit with your very first commit.
Why fixing your own commit later? Just do it right in the first commit ;-)